jwk.go

```diff
@@ -56,6 +56,67 @@ type JsonWebKey struct {
 	Algorithm string
 }
 
+// rawJsonWebKeySet represents a JWK Set with raw members.
+type rawJsonWebKeySet struct {
+	Keys []json.RawMessage `json:"keys"`
```

Comment:
If `JsonWebKey` implements `MarshalJSON` and `UnmarshalJSON`, would it not be possible to put `[]JsonWebKey` in here and have the `json` package parse it directly?